### PR TITLE
Unskip repro for 17339

### PR DESCRIPTION
**Pull request by diogormendes**
### Status
PENDING CI / PENDING REVIEW

### What does this PR accomplish?
- Unskip #17339

### How to test this manually?
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/collections/collections.cy.spec.js`
- All tests should pass

### Additional notes:
- The click now happens on the `chevron` instead of the collection itself.

### Screenshots:

Screen Shot 2022-04-06 at 12 27 30
pful

Stress Test: https://github.com/metabase/metabase/actions/runs/2104067911

### CODE REVIEW

**Comment by diogormendes** [comment]
@nemanjaglumac @flamber I would appreciate the feedback here. 
With the changes I've made we are not doing necessarily what is on the issue ticket (click on the name for expand / collapse), but rather, using the chevron for it as the output of a discussion I had on a Slack channel.

Questions:

1-) Should we keep the way I've changed? If so, I think is better to rename and remove the issue relation
2-) Still keep the changes and leave the relation for historical proposals.
3-) Test the behavior that clicking on the name shouldn't collapse/expand the collection.

File: frontend/test/metabase/scenarios/collections/collections.cy.spec.js [hunk 1]
```diff
@@ -132,17 +132,15 @@ describe("scenarios > collection_defaults", () => {
         //
       });
 
-      it.skip("should expand/collapse collection tree by clicking on parent collection name (metabse#17339)", () => {
+      it("should expand/collapse collection tree by clicking on chevron)", () => {
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/collections/collections.cy.spec.js:135]: Was there a specific reason why you removed the issue reference from the test title?

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/collections/collections.cy.spec.js:135]: Also, "on parent collection name" was intentional. It's not just the chevron that triggers the action. The title is now a bit misleading.

Comment (diogormendes) [reply frontend/test/metabase/scenarios/collections/collections.cy.spec.js:135]: Did you see it here ? https://github.com/metabase/metabase/pull/21489#issuecomment-1090613496 

As I understood, the behavior changed. 

And I rename it following https://github.com/metabase/metabase/pull/21489#issuecomment-1090666673

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/collections/collections.cy.spec.js:135]: I missed that comment previously.
Just opened the local dev and I don't see anything changed. When I click on the parent collection name, it still opens and collapses.

Comment (diogormendes) [reply frontend/test/metabase/scenarios/collections/collections.cy.spec.js:135]: NP. 

When you click on the collection it opens, but to collapse you need to click twice. One click is to select only.

**Review by nemanjaglumac — CHANGES_REQUESTED** [review]
Looking at [Anton's comment](https://github.com/metabase/metabase/issues/17339#issuecomment-1090206251).

Even if that change was intentional, the updated test doesn't reflect that behavior. I'd say this is a regression, but waiting on Maz's comment on that.

If it proves to be a regression - this should be left as skipped and the related issue should be reopened.

If Maz says that "clicking two times to expand/collapse" is the new norm/desired behavior, this test should be expanded with the additional click, rather than clicking on chevrons.

**Pull request closed without merging**